vector/src/main/res/values/donottranslate.xml

```diff
@@ -42,4 +42,6 @@
     <string name="ftue_auth_email_verification_subtitle">To confirm your email address, tap the button in the email we just sent to %s</string>
     <string name="ftue_auth_email_verification_footer">Did not receive an email?</string>
     <string name="ftue_auth_email_resend_email">Resend email</string>
+
+    <string name="location_map_view_copyright">© MapTiler © OpenStreetMap contributors</string>
```

Comment:
Can you add `translatable="false"` please?

Reply:
I didn't know that, done.